apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/geth/call.ex

```
@@ -395,7 +395,7 @@ defmodule EthereumJSONRPC.Geth.Call do
     }
     |> put_if_present(params, [
       {"error", :error},
-      {"createdContractAddressHash", :created_contract_address_hash},
+      {"createdContractAddressHash", :created_contract_address_hash, %{validation: :address_hash}},
```

It seems that the problem only exists for "create" traces that have reverted. Since `createdContractAddressHash` and `createdContractCode` should be specified only for successful traces, maybe it makes sense to explicitly drop them in case the trace has failed. In this case we would not need a "validation" logic.

It will raise an error in case JSON RPC does not output created contract information for successful traces, the current validation solution, however will silently ignore such JSON RPC issue.

---

I added parsing function clause for failed internal transaction, but would like to keep validation logic in the case, if something wrong will come from the node. Not having a validation approach prevents to insert all internal transactions for affected block. In order to fix "silence" issue, I added warning, if the value didn't pass validation. All changes are in https://github.com/blockscout/blockscout/pull/10443/commits/faceb6cb853c8e78bf94fba76cc160378c52ac72 commit.